docs/source/guides/data-scientist/00-connect-to-domain.rst

```diff
@@ -0,0 +1,170 @@
+
+#. **Obtain** the Login Credentials
+#. **Login** to your Server
+#. **Explore** the User HomePage
+
+.. _step-ds-1:
+
+Step 1: Obtain the Login Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To utilize the ``privacy-enhancing`` features and play around with different ``privacy budgets``, 
+Data Scientists must first get their login ``credentials``, using which they can login into their 
+organization's private data servers. Here are a few points to follow to get your login credentials:
+
+* If your account was created by the Domain Owners within your organization, you could ask them to 
+  securely ``share`` the credentials with you. If you do not have a domain owner, you can ``create`` 
+  one locally following the tutorials starting here: `Deploy your own Domain Server <../data-owner/00-deploy-domain.html>`_.
+  
+* You can also signup or create an account on a Domain node if you have access to the ``URL`` to the Domain. 
+  To ``register`` yourself to the Domain, you need to run the following code:
+
+.. WARNING::
+   Change the default username and password below to a more secure and private combination of your preference.
+
+::
+
+   '''
+   Name: Name of the Data Scientist
+   Email: Email of the Data Scientist
+   Password: A secured password to log into the Domain
+   Url: Url to the domain node.
+   Port: Port number
+   '''
+
+   In:
+
+   # run this cell
+   import syft as sy
+   domain_client = sy.register(
+      name="Alice",
+      email="alice@email.com",
+      password="supersecurepassword",
+      url="localhost",
+      port=8081
+   )
+
+.. note::
+   By default, the role assigned to the registered user is of a Data Scientist, and the assigned privacy budget is 0.
+
+
+Step 2: Login to your Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have your login credentials, you can ask your Domain Owner to provide you with the link to the ``URL`` to access your Domain.
+
+.. WARNING:: 
+   If you do not have a domain owner, you can ``create`` one locally following the tutorials 
+   starting here: `Deploy your own Domain Server <../data-owner/00-deploy-domain.html>`_.
+
+Using the link, you can visit the Data Scientists ``login page``, as shown in the image below.
+
+|00-connect-to-domain-01|
+
+The ``Email`` and ``Password`` are your login credentials which you already have if you executed :ref:`Step 1 <step-ds-1>` above.
+
+Once we're ``logged`` in, you can move to the next section, which ``explores`` the user homepage.
+
+
+Step 3: Explore the User HomePage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you login, the first page seen by the Data Scientists is the ``User`` homepage.
+
+When on this page, you can ``create`` an account for other team members within your organization and 
+check the requests for ``pending`` users and ``denied`` users. Also, it gives you information like privacy budget 
+allocated, date added, email, and the name of other users in a table-like format.
+
+|00-connect-to-domain-02|
```

Comment:
@abhiwalia15 we can skip adding the Permissions, User Create and Networks sections for a Data Scientist.
A Data scientist would ideally live in the notebooks. From an UI perspective, we can just explain the details of Data Requests and Budget Requests, remaining sections could be removed. cc @Kiaka007